bridge/RtpMap.h

```diff
@@ -66,6 +66,8 @@ struct RtpMap
 
     RtpMap(const RtpMap& rtpMap) = default;
 
+    bool isEmpty() const { return format == Format::EMPTY; }
+
     Format format;
     uint16_t payloadType;
```

Comment:
I wonder if this should be uint8_t. Or do we expect payload types outside that range? Even payload types > 127 are not valid.

Reply:
I changed to uin8_t